websocket/polygon.go

```diff
@@ -128,7 +128,18 @@ func (c *Client) connect() error {
 	return nil
 }
 
+func (c *Client) checkPrefix(topic Topic) {
```

Comment:
given the name, i'd expect this interface to look like `checkPrefix(topic Topic) bool` but i get that this is actually supposed to set missing topic prefixes. maybe `setTopic` or `setPrefix` would be more clear?